app/views/idv/shared/_mini_logo.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= image_tag(
+      asset_url('logo.svg'),
```

Comment:
You probably (sadly) need to use a PNG version for emails:

https://www.caniemail.com/features/html-svg/